apps/docs/pages/guides/auth/phone-login/twilio.mdx

```diff
@@ -347,8 +347,30 @@ let { data, error } = await supabase.auth.verifyOtp({
 })
 ```
 
+## Twilio Verify
```

Comment:
I would prefer if this is the first section. Twilio Verify is what most people should use, so it should be front-and-center. Maybe also re-order the dropdown in the dashboard to have Twilio Verify be first.